### PR TITLE
feat(localization): Add localization support

### PR DESCRIPTION
**Pull request by GordonSmith**
Added zh translations

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@wangkx please review the zh translations.

File: package.nls.zh.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    "Create ECL Launch": "建立ECL启动",
+    "Creating Archive": "建立档案",
+    "Creating Workunit": "建立Workunit",
+    "Current document is not a 'Dashy' file": "当前文件不是'Dashy'文件",
+    "Current document is not a 'ECL Markdown' file": "当前文件不是'ECL Markdown'文件",
+    "Dashy Library Path (libraryLocation === \"localPath\")": "Dashy库路径(libraryLocation === \"localPath\")",
+    "Dashy library location (bundled, latest, localPath)": "Dashy库位置(bundled, latest, localPath)",
+    "Debug level logging (requires restart)": "调试级别日志记录（需要重启）",
+    "Default launch configuration": "默认启动配置",
+    "Default timeout (secs)": "默认超时设定",
```
Comment (wangkx) [inline package.nls.zh.json:33]: "默认超时设定(秒)"

File: package.nls.zh.json [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    "Compile ECL on the HPCC Platform": "在HPCC平台编译ECL",
+    "Connection failed": "连接失败",
+    "Create ECL Launch": "建立ECL启动",
+    "Creating Archive": "建立档案",
+    "Creating Workunit": "建立Workunit",
+    "Current document is not a 'Dashy' file": "当前文件不是'Dashy'文件",
+    "Current document is not a 'ECL Markdown' file": "当前文件不是'ECL Markdown'文件",
+    "Dashy Library Path (libraryLocation === \"localPath\")": "Dashy库路径(libraryLocation === \"localPath\")",
+    "Dashy library location (bundled, latest, localPath)": "Dashy库位置(bundled, latest, localPath)",
+    "Debug level logging (requires restart)": "调试级别日志记录（需要重启）",
```
Comment (wangkx) [inline package.nls.zh.json:31]: 打开或关闭调试日志记录（需要重启）

**Review by wangkx — COMMENTED** [review]
@GordonSmith I suggest 2 changes in Chinese translations.

**Pull request merged**